### PR TITLE
feat: update some internal links to point to new fair page structure

### PR DESCRIPTION
**Pull request by mzikherman**
There's also these! I'm not 100% sure where these get used (if at all). For instance, the fair listings page is powered by Metaphysics (via `href`), and at least one place this _is_ being referenced, is likely orphaned code (something to do with search results in coffeescript, AFAIK everything related to search is new React(ion) code).

### CODE REVIEW

File: src/mobile/test/models/fair.test.js [hunk 1]
```diff
@@ -28,9 +28,7 @@ describe("Fair", function () {
 
   describe("#href", () =>
     it("returns a client link using the organizer profile id", function () {
-      return this.fair
-        .href()
-        .should.equal(`/${this.fair.get("organizer").profile_id}`)
+      return this.fair.href().should.equal("/fair/armory-show-2013")
```
Comment (anandaroop) [inline src/mobile/test/models/fair.test.js:31]: The test description on L30 could use an update.

File: src/mobile/test/models/fair.test.js [hunk 2]
```diff
@@ -28,9 +28,7 @@ describe("Fair", function () {
 
   describe("#href", () =>
     it("returns a client link using the organizer profile id", function () {
```
Comment (anandaroop) [inline src/mobile/test/models/fair.test.js:30]:
```suggestion
    it("returns a client link using the id", function () {
```

**Review by anandaroop — COMMENTED** [review]
Just a tiny suggestion. Other than that LGTM.

**Comment by mzikherman** [comment]
merging in! (after several CI rebuilds 😅 )

**Pull request merged**